app/models/manageiq/providers/redhat/infra_manager/vm.rb

```diff
@@ -47,8 +47,4 @@ def self.calculate_power_state(raw_power_state)
   def validate_publish
     validate_unsupported("Publish VM")
   end
-
-  def validate_clone
```

Comment:
1) hm about supports_not :clone, I thought that if you add :clone to QUERYABLE_FEATURES it automatically included supports_not
2) I believe `cloneable? #> true` useless method here because `validate_clone` under `ManageIQ::Providers::Redhat::InfraManager::Vm` just override `validate_clone` under `module Vm::Operations::Lifecycle`